packages/core/src/application_ref.ts

```diff
@@ -818,6 +819,7 @@ export class ApplicationRef {
   private _runningTick: boolean = false;
   private _destroyed = false;
   private _destroyListeners: Array<() => void> = [];
+  private _afterRenderHooksManager: AfterRenderHooksManager|null = inject(AfterRenderHooksManager);
```

Comment:
We could also drop typings here, the type should be inferred (also it can't be `null` in our case, since it'd `providedIn: 'root'`):
```suggestion
  private _afterRenderHooksManager = inject(AfterRenderHooksManager);
```